packages/react-core/src/components/DualListSelector/DualListSelectorControlsWrapper.tsx

```diff
@@ -76,7 +76,7 @@ DualListSelectorControlsWrapperBase.displayName = 'DualListSelectorControlsWrapp
 
 export const DualListSelectorControlsWrapper = React.forwardRef(
   (props: DualListSelectorControlsWrapperProps, ref: React.Ref<HTMLDivElement>) => (
-    <DualListSelectorControlsWrapperBase innerRef={ref as React.MutableRefObject<any>} {...props} />
+    <DualListSelectorControlsWrapperBase innerRef={ref as React.MutableRefObject<any>} {...props} role="region" />
```

Comment:
For the role, would "region" or "group" be better in this context? Currently the selector controls appear in VO's rotor under "landmarks", and wasn't sure if that may have been part of the intent or not.

Reply:
ooo good thought